client/src/components/ReviewTab/ReviewTab.tsx

```diff
@@ -1,35 +1,69 @@
-import { Box, Typography, Grid } from '@material-ui/core';
+import { Box, Typography, Grid, Paper } from '@material-ui/core';
 import ReviewTabForm from './ReviewTabForm/ReviewTabForm';
 import { Review } from '../../interface/User';
 import useStyles from './useStyles';
-import ArtistCarousel from '../ArtistCarousel/ArtistCarousel';
+import ReviewCarousel from '../ReviewCarousel/ReviewCarousel';
 import Switch from '@material-ui/core/Switch';
 import { ChangeEvent, useState } from 'react';
+import { useAuth } from '../../context/useAuthContext';
+import { useSnackBar } from '../../context/useSnackbarContext';
+import { createReviews, getReviews } from '../../helpers/APICalls/review';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import { useEffect } from 'react';
+import Avatar from '@material-ui/core/Avatar';
+import Rating from '@material-ui/lab/Rating';
 
-export default function PersonalInfo(): JSX.Element {
+type Props = {
+  artistId: string;
+};
+
+export default function ReviewTab({ artistId }: Props): JSX.Element {
   const classes = useStyles();
   const [checked, setChecked] = useState(false);
+  const { loggedInUser } = useAuth();
+  const { updateSnackBarMessage } = useSnackBar();
+  const [reviews, setReviews] = useState<Review[]>([]);
 
   const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
     setChecked(event.target.checked);
   };
-
-  const handleSubmit = ({ rating, text }: Review) => {
-    console.log('rating is', rating);
-    console.log('text is', text);
+  const handleSubmit = async ({ rating, text }: Review) => {
+    const review = {
+      artistId,
+      rating,
+      text,
+    };
+    const result = await createReviews(review);
+    if (result.error) {
+      updateSnackBarMessage(result.error);
+    } else {
+      updateSnackBarMessage('Review submitted successfully');
+      const updatedRev = reviews ? [...reviews, result] : [result];
+      setReviews(updatedRev);
+    }
   };
 
-  return (
-    <Box textAlign="center">
+  useEffect(() => {
+    async function reviews() {
+      const reviews = await getReviews(artistId);
+      if (reviews !== null) {
+        setReviews(reviews);
+      } else {
+        new Error('Could Not Get Artist Info');
+      }
+    }
+    reviews();
```

Comment:
Please give this function a more descriptive name :)

Reply:
fixing in the next commit :) thank you